index.js

```diff
@@ -74,6 +53,16 @@ class BabelCompiler {
     this.options.filename = file.path;
     this.options.sourceFileName = file.path;
 
+    if (!this.options.presets) {
+      const babelConfig = new babel.OptionManager().init(this.options);
+      console.log(babelConfig);
+      const hasConfig = babelConfig.presets && babelConfig.presets.length > 0;
+
+      if (!hasConfig) {
+        this.options.presets = ['latest'];
```

Comment:
could be

```js
const opts = {targets: {}};
if (process.env.NODE_ENV === 'production') {
  opts.targets.uglify = true;
}
this.options.presets = ['env', opts];
```
Also good to detect uglify in package.json and add it to condition above.